.changeset/rare-plants-itch.md

```diff
@@ -0,0 +1,6 @@
+---
+'@nl-design-system-community/editor': minor
+'@nl-design-system-community/editor-react': minor
+---
+
+Initial alpha release of `@nl-design-system-community/editor` and `@nl-design-system-community/editor-react`.
```

Comment:
De changelog tekst komt terecht bij beide packages. Daarin is het niet helpful als er wordt gesproken over een andere package, "Initial npm release of package." zou volstaan.